site/content/blog/edit-a-website-locally.md

```diff
@@ -0,0 +1,250 @@
+- Gain a deeper understanding of working with Git and GitHub Desktop.
+- Learn how to clone your website's repository to your computer.
+- Learn how to edit content using Obsidian and what are the benefits of it.
+- Learn how to commit (save) your changes locally and push them back to the GitHub repository.
+
+Below is a screenshot of how the final website will look like:
+
+![[tutorial-2-result.png]]
+
+Let's start by understanding why using GitHub UI as we did in tutorial 1 is not always a good choice.
+
+## What is Markdown and why use Obsidian for editing it in Flowershow-based websites
+
+While editing on the GitHub UI is convenient, there are certain limitations and drawbacks to consider. For example, you can't work offline, can't add or edit multiple files simultaneously, and the GitHub UI's preview doesn't render all the Markdown syntax elements that are supported by Flowershow-based websites.
+
+Ok, but, what exactly is Markdown?
+
+[Markdown](https://en.wikipedia.org/wiki/Markdown) is a simple and intuitive syntax, that allows you to structure your text documents. It's widely used for creating content on the web. As Wikipedia puts it:
+
+> *Markdown is a lightweight [**markup language**](https://en.wikipedia.org/wiki/Markup_language) for creating [**formatted text**](https://en.wikipedia.org/wiki/Formatted_text) using a **plain-text editor**.*
+
+The key term here is "markup language". In simple terms, it is a way to annotate or "mark up" a plain text document with symbols that describe how the document is structured and so, how it should be understood, processed, or displayed. These tags can tell a computer program that supports this syntax (e.g. a website) how to format the rendered text, for example: which words should be bold or italic, where to insert images, when to start new paragraphs, how to create tables, and so forth. It's important to note, that even though Markdown symbols change how the text is displayed when it's rendered by a Markdown-compatible viewer, the underlying document in still just a plain text. For instance, if you want to create a heading in Markdown, you use the "#" symbol before your heading text, like this:
+
+```md
+# This is a Heading
+```
+
+BUT, there is no single version of Markdown. It comes in different "flavours", with CommonMark and GitHub Flavoured Markdown (GFM) being the most popular ones. And different tools supporting Markdown may use their own specific versions of Markdown. The two tools relevant in the context of this guide are Obsidian and Flowershow.
+
+Obsidian supports an extended version of Markdown that includes majority of elements from CommonMark and GFM, while also introducing its own unique features, like [wiki-links](https://help.obsidian.md/Linking+notes+and+files/Internal+links) or [callouts](https://help.obsidian.md/Editing+and+formatting/Callouts). And Flowershow template is Obsidian-compatible, meaning it supports (or aims to support) all of the Obsidian Markdown features.
```

Comment:
> And Flowershow template is Obsidian-compatible,

Perhaps it was meant to be _And Flowershow is Obsidian-compatible_. Just checking.